modules/openapi-generator/src/test/resources/2_0/petstore_plus_unique.json

```diff
@@ -0,0 +1,206 @@
+{
+  "openapi": "3.0.0",
```

Comment:
it's a 3.0 spec. can you please put it under the 3_0 folder instead?

Reply:
Fixed it , Thx